docs/Presentaciones/02-04-2024.md

```diff
@@ -0,0 +1,242 @@
+
+**OPCIONAL, SOLO SI HAY TIEMPO**
+(Además, nos permite concertar las reuniones en cualquier herramienta cómo discord y conectar el google calendar)
+
+## Análisis de la calidad
+
+<p align="center">
+<img src="\img\pptx\codacy.png" alt="qualityCodacy" width="60%" height="60%"/>
+Figura 28. Análisis de la calidad
+</p>
+
+**En este caso, vamos a hablar del análisis generado por codacy para nuestros repositorios de código**. La calidad de código desde el principio ha sido uno de nuestros objetivos, ya que un código de calidad es mantenible, usable y fiable. 
+
+<p align="center">
+<img src="\img\pptx\codacy2.png" alt="qualityCodacy2" width="60%" height="60%"/>
+Figura 29. Gráfica del Sprint 2
+</p>
+
+Aun así, me gustaría mencionar algunas cosas, cómo por ejemplo la gráfica del sprint 2 en la que podemos ver cómo al final del Sprint 1 hubo una etapa en la que el código tenía algunos problemas debido al retraso en las tareas que hizo que no se pudiesen resolver estos problemas antes del sprint 2, aunque podemos ver cómo la semana siguiente se resolvieron, manteniendo la calidad en niveles óptimos.
+
+<p align="center">
+<img src="\img\pptx\coverage.png" alt="Cobertura" width="60%" height="60%"/>
+Figura 30. Cobertura de tests
+</p>
+
+
+Sin embargo, la cobertura de los tests cómo podemos ver ha decaído un poco este segundo sprint, debido a que hemos decidido avanzar todo lo posible con la funcionalidad y testear únicamente el código de valor. Aun así, podemos asegurar que toda la funcionalidad realizada en este segundo sprint está cubierta. 
+
+## Gestión de problemas 
+
+<p align="center">
+<img src="\img\pptx\pmanage.png" alt="Problemas" width="60%" height="60%"/>
+Figura 31. Gestión de problemas
+</p>
+
+**Otro aspecto imporante durante el sprint ha sido la gestión de problemas.** El único problema que hemos tenido ha sido el despliegue de la base de datos con postgresql, después de estudiar las opciones, se decidió hacer la migración a mongoDB. El objetivo de esta solución era reducir los costes y la complejidad del despliegue de la base de datos y tras haber realizado un análisis hemos conseguido reducir el coste aunque hemos tenido que replanificar algunas tareas debido a la migración. Por último, hemos calificado esta solución con un 3,5 sobre 5. **Y os preguntaréis, ¿esta calificación de dónde viene?**
+
+<p align="center">
+<img src="\img\pptx\pmanage2.png" alt="solMetrica" width="60%" height="60%"/>
+Figura 32. Fórmula de calidad de la solución
+</p>
+
+Pues esa puntuación se obtiene mediante esta fórmula en la que tenemos dos factores:
+- El beneificio neto de la solución
+- Los recursos utilizados
+Ambas son rúbricas del 1 al 5 que están detalladas en el docusaurus.
```

Comment:
One more blank line at the end of the list

Reply:
Thank you!